components/Layout.tsx

```diff
@@ -348,9 +355,12 @@ const Footer = () => (
             href='https://json-schema.org/slack'
             className='flex items-center text-white'
           >
-            <img
+            <Image
               src='/img/logos/slack_logo_small-white.svg'
+              width={16}
+              height={16}
               className='w-4 h-4 mr-2'
+              alt='Slack logo'
```

Comment:
For the instances where we are already controlling height and width with CSS, why do we need to also set them explicitly for the image tag=

Reply:
will remove height and class from classname